packages/core/types/Config.tsx

```diff
@@ -86,6 +87,9 @@ export type Config<
       Props[ComponentName]
     >;
   };
+  tags?: {
+    defaultLabel?: string;
+  };
```

Comment:
I think we can make better use of this API-namespace to future proof it

```
  tags: {
    default: {
      label: "Others"
    },
    layout: {
      label: "Layout"
    }
  }
```

We can then create a strict type-link between the tags specified here and the tags available for the components

Reply:
Yeah, I like it. I wonder if it's also worth adding a weight field like so

` tags: {
    default: {
      label: "Others"
      weight: 0
    },
    layout: {
      label: "Layout"
      weight: 2
    }
  }`
  
  That way you could reorder the order that they display?